nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/peaks/PeakListTools.java

```diff
@@ -1497,13 +1497,8 @@ public static List<Object> peakFit(PeakList peakList, Dataset theFile, List<Peak
             }
             firstPeak = false;
         }
-        if ((delays != null) && (delays.length > 0)) {
-            GuessValue gValue = new GuessValue(0.0, -0.5 * globalMax, 0.5 * globalMax, false);
-            guessList.add(0, gValue);
-        } else {
-            GuessValue gValue = new GuessValue(0.0, -0.5 * globalMax, 0.5 * globalMax, false);
-            guessList.add(0, gValue);
-        }
+        GuessValue gValue = new GuessValue(0.0, -0.5 * globalMax, 0.5 * globalMax, false);
+        guessList.add(0, gValue);
```

Comment:
removing the if/else resulted in the variable needed to be defined in the scope, so changed a few more lines below this to fix redeclaring the variable

Reply:
I'd like to avoid reusing variables if we can. You renamed to `gVal` line 1549 which is good, could you do something similar line 1529, or inline the variable here?

Reply:
used inline variables and renamed another reuse of gValue as well